config/locales/en.yml

```diff
@@ -266,7 +266,7 @@ en:
   customer_instructions: "Customer instructions"
   additional_information: "Additional Information"
   connect_app:
-    url: "https://n8n.openfoodnetwork.org.uk/webhook/regen/connect-enterprise"
+    url: "https://n8n.openfoodnetwork.org/webhook/regen/connect-enterprise"
```

Comment:
Why not do this for en_AU with Transifex?

I guess this is quicker, and besides it's now a less-specific URL, so a better example for the default locale :)

Reply:
Oh, and allows for testing 👍